Ffp 32/sh: Add Tag

lpowell2 opened the pull request:
This request is for adding tags to the database. Please be logged in and go to localhost/fumc-fb/registernewtag.php.
PLEASE DON'T APPROVE TO BE MERGED INTO MAIN. Just leave feedback

Jeanneveev commented on addTagForm.php:
```diff
@@ -0,0 +1,19 @@
+        <p>Press submit to add tag.</p>
+        <input type="submit" name="addtag-form" value="Submit">
+    </form>
+        <a class="button cancel" href="index.php" style="margin-top: .5rem">Cancel</a>
```
Should be some check if tag name or id matches any other in the database, and cancel if so

Jeanneveev commented on sql/vms.sql:
```diff
@@ -67,6 +67,28 @@ CREATE TABLE `dbEventVolunteers` (
+-- Table structure for table `dbTags`
+--
+
+CREATE TABLE `dbTags` (
```
Best to change to CREATE TABLE IF NOT EXISTS so that MySQL can import it without causing errors if there's a pre-existing table with that name.
Then, any edits to the table could be put in ALTER TABLE

Jeanneveev reviewed (commented):
Review Complete

lpowell2 committed:
redoing a commit ignore

sarah-harrington committed:
updates, code cleanup

sarah-harrington committed:
Merge branch 'ffp-32/sh' of https://github.com/lpowell2/FUMC-FB into ffp-32/sh

sarah-harrington committed:
Merge branch 'dev' into ffp-32/sh

sarah-harrington committed:
small updates

-removed uneccessary add tag
updated header access levels
created page to display buttons for managing tags

sarah-harrington committed:
header fix
